doc/source/serve/deployment-graph.md

```diff
@@ -492,18 +494,20 @@ with InputNode() as dag_input:
     # Use output of function DeploymentNode in bind()
     dag = combiner.run.bind(
         preprocessed_1, preprocessed_2, dag_input[2]
-    ) 
-    
+    )
+
     # Each serve dag has a driver deployment as ingress that can be user provided.
-    serve_dag = DAGDriver.options(route_prefix="/my-dag").bind(dag)
+    serve_dag = DAGDriver.options(route_prefix="/my-dag", num_replicas=2).bind(
```

Comment:
verified locally running this example works end to end. we should convert this to a Python file and run in CI (but in a separate PR)